src/components/Dreps/DrepsList/index.tsx

```diff
@@ -97,25 +97,29 @@ const DrepsList: React.FC = () => {
         r.anchorUrl != null ? (
           <CustomTooltip title={r.anchorUrl ? r.anchorUrl : undefined} sx={{ width: 150 }}>
             <Box
+              padding={"6px 0px"}
               data-testid="drepList.anchorLinkValue"
               component={Button}
               textTransform={"lowercase"}
               fontWeight={400}
               display={(r.anchorUrl || "").length > 20 ? "inline-block" : "inline"}
-              width={"150px"}
+              width={(r.anchorUrl || "").length > 20 ? "150px" : "fit-content"}
+              minWidth={0}
               textOverflow={"ellipsis"}
               whiteSpace={"nowrap"}
               overflow={"hidden"}
               color={(theme) => `${theme.palette.primary.main} !important`}
-              onClick={() => setMetadataUrl(r.anchorUrl)}
+              onClick={() => {
+                r.anchorUrl.includes("http") ? setMetadataUrl(r.anchorUrl) : setMetadataUrl(`//${r.anchorUrl}`);
```

Comment:
What does adding `//` in front of the anchorURL do? So I know what it's doing, but why do we need it? Asking just out of curiosity.